docs/sbpy/data/orbit.rst

```diff
@@ -153,14 +162,25 @@ One can also specify the planet with respect to which the Tisserand parameter
 is calculated with optional parameter `planet`.  It also allows multiple
 planet to be specified simultaneously:
 
+.. .. doctest-requires:: astroquery
 .. doctest-remote-data::
```

Comment:
Should this remote-data derivative be removed if the test is now using hardcoded orbital elements?  Same for line 194.

Reply:
I'd like to keep them here as a reminder until the doctest-requires issue is addressed.